tests/AudioWaveformer.cpp

```diff
@@ -130,3 +130,75 @@ TEST_CASE( "Normalize & scale waveform data piano.wav", "[libopenshot][audiowave
     // Clean up
     r.Close();
 }
+
+TEST_CASE( "Extract waveform from image (no audio)", "[libopenshot][audiowaveformer]" )
+{
+    // Create a reader
+    std::stringstream path;
+    path << TEST_MEDIA_PATH << "front.png";
+    FFmpegReader r(path.str());
+
+    // Create AudioWaveformer and extract a smaller "average" sample set of audio data
+    AudioWaveformer waveformer(&r);
+    AudioWaveformData waveform = waveformer.ExtractSamples(-1, 20, false);
+
+    CHECK(waveform.rms_samples.size() == 0);
+    CHECK(waveform.max_samples.size() == 0);
+
+    // Clean up
+    r.Close();
+}
+
+TEST_CASE( "AudioWaveformData struct methods", "[libopenshot][audiowaveformer]" )
+{
+    // Create a reader
+    AudioWaveformData waveform;
```

Comment:
Test all of the methods on our `AudioWaveformData` object - mostly to confirm that my new "zero()" method actually works.